EstateAdministrationUI.Testing/TestData.cs

```diff
@@ -0,0 +1,303 @@
+﻿namespace EstateAdministrationUI.Testing
+{
+    using System;
+    using System.Collections.Generic;
+    using Areas.Estate.Models;
+    using BusinessLogic.Models;
+    using EstateManagement.DataTransferObjects.Responses;
+
+    public class TestData
+    {
+        public static Guid EstateId = Guid.Parse("FC657462-2620-4B35-B744-1BD7E60C1748");
+
+        public static String EstateName = "Test Estate 1";
+
+        public static String OperatorName = "Test Operator 1";
+
+        public static Guid OperatorId = Guid.Parse("DECA8293-F045-41C5-A2F7-30F2792FD273");
+
+        public static Boolean RequireCustomMerchantNumber = true;
+
+        public static Boolean RequireCustomTerminalNumber = true;
+
+        public static String EmailAddress = "estateuser1@testestate1.co.uk";
```

Comment:
Codacy found an issue: [Change the visibility of 'EmailAddress' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5756318)